pydra/tasks/mrtrix3/__init__.py

```diff
@@ -1,6 +1,10 @@
 """
 >>> import pydra
+<<<<<<< HEAD
+>>> import pydra.tasks.MRtrix3
+=======
 >>> import pydra.tasks.mrtrix3
+>>>>>>> 011f7c13c16e25bcaeb0d9224e490925d45f1550
```

Comment:
```suggestion
>>> import pydra.tasks.mrtrix3
```

Reply:
Yup caught this after the push, didn't SAVE my resolved conflict -__-"